prologin/users/oidc_authz.py

```diff
@@ -0,0 +1,13 @@
+from django.core.exceptions import PermissionDenied, ObjectDoesNotExist
+from .models import OpenIDClientPolicy
+
+def authorize(request, user, client):
+    policy = None
+    try:
+        policy = OpenIDClientPolicy.objects.get(pk=client.pk)
```

Comment:
```suggestion
        policy = OpenIDClientPolicy.objects.get(openid_client=client)
```
c'est pas plutôt un truc comme ça ?

Reply:
oui c'est plus clair comme ça